apps/open-swe/src/graphs/planner/nodes/proposed-plan.ts

```diff
@@ -85,9 +86,16 @@ async function startProgrammerRun(input: {
   newMessages?: BaseMessage[];
 }) {
   const { runInput, state, config, newMessages } = input;
-  const langGraphClient = createLangGraphClient({
-    defaultHeaders: getDefaultHeaders(config),
-  });
+  let langGraphClient;
+  if (isLocalMode(config)) {
+    langGraphClient = createLangGraphClient({
+      defaultHeaders: { "x-local-mode": "true" },
+    });
+  } else {
+    langGraphClient = createLangGraphClient({
+      defaultHeaders: getDefaultHeaders(config),
+    });
+  }
```

Comment:
can make this more concise:
```suggestion
  const langGraphClient = createLangGraphClient({
    defaultHeaders: isLocalMode(config)
      ? { "x-local-mode": "true" }
      : getDefaultHeaders(config),
  });
```